.github/workflows/test.yml

```diff
@@ -15,6 +15,7 @@ jobs:
         os: [ubuntu-latest, macos-latest]
         python-version: ['3.8', '3.9']
         deps: [newest-deps]
+        tree-sitter-bash: [system, bundled]
```

Comment:
Better suggestions for names are welcome :-) the value "system" doesn't really mean much when looking at the build overview. A key like "tree-sitter-bash-sys" or something would probably be better.